### PR TITLE
Desktop Version Pull Request

### PR DESCRIPTION
**Pull request by Habtamu210**
In this project:
- Media query is used to make changes from the mobile version to the desktop version

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -406,3 +536,305 @@ footer > hr {
...
+    grid-template-columns: 1fr 1fr 1fr;
+    grid-template-rows: 2;
+    border-radius: 0;
+    row-gap: 0;
+    column-gap: 12px;
+    margin-top: 4px;
+    margin-bottom: 24px;
+    margin-left: 30px;
+    margin-right: 0;
+  }
```
Comment (pinkmoon25) [inline style.css]: Great job implementing the desktop version :+1: , The project cards are not how they are in the original figma design, please make sure they are setup correctly, your design :point_down: 
image
Figma design :point_down: 
image
Please try to use flex for individual cards layout and make it as close to the figma design as possible.

File: style.css [hunk 2]
```diff
@@ -406,3 +536,305 @@ footer > hr {
...
+    margin-left: 80px;
+  }
+
+  .about-me > .about-me-main-1 {
+    background-position: center;
+    background-repeat: no-repeat;
+    height: 652px;
+    width: 652px;
+  }
+
```
Comment (pinkmoon25) [inline style.css]: please try to align your About myself heading to the left following the design from figma, your design :point_down: 
image
 figma design :point_down: 
image

File: index.html [hunk 3]
```diff
@@ -7,227 +7,242 @@
...
+        <h3 class="fhead">Skills</h3>
+      </div>
+      <ul class="language-list framework">
+        <li>Codekit</li> 
+        <li>Github</li>
+        <li>Codepen</li>
+        <li>Gitlab</li>
+        <li>Terminal</li>
+      </ul>
+    </li>
```
Comment (pinkmoon25) [inline index.html:214]: please try to fix the font-style and size for your language section, your design :point_down: 
image
figma design :point_down: 
image

**Review by pinkmoon25 — CHANGES_REQUESTED** [review]
Hi @Habtamu210 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
linter checks :heavy_check_mark: 
professional readme :heavy_check_mark: 
follows design template :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 4]
```diff
@@ -7,227 +7,242 @@
...
+            <p class="project-description">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+            <ul class="languages">
+              <li class="langn">css</li>
+              <li class="langn">html</li>
+              <li class="langn">bootstrap</li>
+              <li class="langn">Ruby</li>
+            </ul>
+            <button class="project-button">See Project</button>
+          </div>
+        </div>
```
Comment (christianbiring1) [inline index.html:77]: [OPTIONAL]
- [ ] You may also consider using the same image in your cards as the ones provided on the template in Figma.

**Review by christianbiring1 — APPROVED** [review]
Hi @Habtamu210 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
# STATUS: APPROVED :tada: 
nicee

## Highlights
- [ ] All linters checks are green :heavy_check_mark: 
- [ ] Descriptive PR title and summary
- [ ] Good commit messages :+1: 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**